scripts/patch_metal_spec.py

```diff
@@ -212,6 +217,30 @@ def loadYaml(fn):
     "$ref": "#/components/schemas/Href"
 }
 
+# FIX 10. 
```

Comment:
What is generated with the existing spec contents?  The `SSHKeyCreateInput` request body is [defined in the fetched spec](https://github.com/equinix-labs/metal-python/blob/spec_update_02052023/oas3.fetched/components/requestBodies/SSHKeyCreateInput.yaml), and its content matches what is inlined here.

Reply:
What is the error message from the generator?  I don't think we've run into an issue on this part of the spec in other clients.

Reply:
I checked out this branch and poked around, and I see that [line 144 of this script](https://github.com/equinix-labs/metal-python/pull/29/files/1cffce7da64571edd93a6b86565eb140d0c23fac#diff-99974451618f89f324e430e5e647f8180ae7912d3ba9d0be6cd63ba8aba211b8L144) is deleting any and all `requestBodies`.  Removing that line and allowing the `requestBodies` to end up in the stitched spec fixes the problem without this patch.

Reply:
@ctreatma I see. I removed the request bodies because none of them were necessary in the basic set of resources that I used in the initial version. Now that I included SSH keys, it's needed and I misinterpreted the error and filled the missing ref wrong. It was my fault, sorry.

I'm thinking how to resolve this. I am thinking to not prune the API spec at all, and generate all the code. I pruned the spec initially because it was really slow to generate all the code with the generators at my disposal. The latest stable python-nextgen is quite fast, so the duration of the code generating is not a problem anymore.

Do you agree? would you like me to do a new PR where I include code generated from the whole (current) spec? Or should I do it in this PR?